apps/src/lab2/levelEditors/aichatSettings/AiCustomizationFieldSection.tsx

```diff
@@ -0,0 +1,62 @@
+import classNames from 'classnames';
+import React, {useContext} from 'react';
+
+import {AiCustomizations} from '@cdo/apps/aichat/types';
+import {BodyFourText} from '@cdo/apps/componentLibrary/typography';
+
+import {UpdateContext} from './UpdateContext';
+import VisibilityDropdown from './VisibilityDropdown';
+
+import moduleStyles from './edit-aichat-settings.module.scss';
+
+interface AiCustomizationFieldSectionProps {
+  fieldName: keyof AiCustomizations;
+  description?: string;
+  inputType: 'text' | 'number' | 'textarea' | 'custom';
+  inputNode?: React.ReactNode;
+  min?: number;
+  max?: number;
+  step?: number;
+}
+
+const AiCustomizationFieldSection: React.FunctionComponent<
```

Comment:
I pulled out from `CollapsibleSection` so I could use for the student system prompt within a collapsible section that also included the level system prompt.
I renamed since only used for AiCustomization fields.